src/backgrounds/StarryBackground.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * @overview A module that initializes the Starry Background parallax effect
+ * @module StarryBackground.js
+*/
+
+export const StarryBackground = {
+
+  /**
+   * Finds page modules with the Starry Background selector and triggers setup
+   * @returns {void}
+  */
+  init() {
+    this.StarryBackgroundEls = document.querySelectorAll(`.background--starry`);
```

Comment:
I figured since it's a module that's being defined in our core components, it is being authored explicitly with certain classnames. The `.background--starry` class name is the foundation necessary for the background to even show up, but I guess it wouldn't be harmful to add a separate JS class to be strict about our conventions if that's preferred